### DISCUSSION TITLE
VS 2022 - Inline Pop Up Covers The Suggestion

### DISCUSSION THREAD

**Discussion by BFischTech** [community/community#17864] [Copilot Conversations]
edit: I'm wondering if this might be related to intelliCode?

I haven't been able to find a discussion on this after looking so thought I would ask here.

Is anyone else having an issue where the actual suggestion text box is covering up the suggestion?  Also I can't get it to suggest anything when I use a plainly worded comment (ie: //write a solution for fizzbuzz), and haven't found anything about that either.

Is there a known solution to this?

Thanks!

![image](https://user-images.githubusercontent.com/90640795/171863715-0a97642c-b890-4071-a9ab-87188f0b1a5f.png)
![image](https://user-images.githubusercontent.com/90640795/171863852-a70bee4c-3be7-432f-84fa-c2362971f83f.png)

**Comment by ProJoeSoft** [comment]
Same issue in VS 17.2.3 Intellisense is impossible to use when setting up the usings sometimes. Also have Resharper. Maybe they're not playing well together.
![image](https://user-images.githubusercontent.com/100368598/171906931-e7b93bbf-5e2f-4957-8884-da3efa8a305f.png)

**Reply by BFischTech** [reply]
Sounds like someone else is having the issue with ReSharper too, I'm currently not using that extension (although I probably should be tbh) so it may be unrelated but a good place to start I'd say.  Maybe the fix suggested by the russia user will work for you.

**Comment by daniellovell** [comment]
I can also corroborate this report. Something in the past week has broken Copilot in VS2022. Please see the image below: 

![image](https://user-images.githubusercontent.com/6250953/172736833-05b46a56-03b3-4f39-bfdc-6f31ba33810b.png)

> GitHub Copilot 1.27.0.0 (v1.27.0.0@f7ffe8ad5)
> GitHub Copilot Agent v1.27.0

**Reply by daniellovell** [reply]
Fixed by updating VS2022 
"Help > Check for Updates"

**Reply by DanzigGLT** [reply]
So you got it fixed??? I updated VS2022 and Copilot plugin several times, no change.

**Reply by daniellovell** [reply]
No actually. It fixed my home setup, but not my setup in the office (same Visual Studio/Copilot version on both machines). Thankfully the prompt disappears for the rest of the session for me after I hit ESC on the suggestion, but its obviously bugged.

**Comment by DanzigGLT** [comment]
@daniellovell informed me about my duplicate entry, I did not find this one. So: Same problem, two different installations (CE/Pro) and both with C++ Workload installed. Started with V1.23.0.0 IIRC, worked before. Here are enough example pictures, don't need to add mine.

**Comment by jasonswearingen** [comment]
I am also having this problem on VS2022 (v`17.2.4`).    Updating to `17.2.5` (latest) did not help.

This pretty much makes github copilot unusable.

Some notes:
- I do not have Resharper installed
- I reset VS to default settings by using `devenv.exe /ResetSettings` and `devenv.exe /ResetUserData`
- I only have ASP.NET and web dev workload installed (No C++)
- I did a "repair" via the Visual Studio Installer
- I don't have any other extensions installed

**Reply by daniellovell** [reply]
I have this issue with the latest 17.2.5 VS2022 on one machine, but not on another machine with the identical versions and extensions. This issue has been active for ~3 weeks now and remains unaddressed, I'm beginning to think this is not the appropriate place to report bugs?

**Comment by BFischTech** [comment]
Weird that it's on one machine and not the other, they can't actually be
identical if that's the case.  At least I think so, right?

It's just low priority I'm sure Daniel and it sounds like it's a weird
little quirk that may be hard to reproduce especially if it can exist with
the same set up like you said.  I haven't fixed it for myself but I also am
getting ready to work on some client stuff sooner or later on a work
computer so I am spending most my time without it anyways.

**Comment by ScyMarshall** [comment]
Hi guys, did there anything happen? I also have that crap in my vs2022!
